src/core/utils/index.js

```diff
@@ -123,6 +123,24 @@ export function hex2string(hex)
     return `#${hex}`;
 }
 
+/**
+ * Converts an integer to closest even integer.
+ *
+ * @memberof PIXI.utils
+ * @function expandToEven
+ * @param {number} input - the integer to convert
+ * @return {number} even integer.
+ */
+export function expandToEven(input)
+{
+    if (input & 1)
+    {
+        input++;
```

Comment:
would we want to put a Math.ceil around the input to ensure that any decimal places are also rounded up to whole integers?

Reply:
In all cases function used, values is rounded already. But ok, i added Math.ceil.